recipe/meta.yaml

```diff
@@ -24,12 +24,14 @@ requirements:
   build:
     - {{ compiler("c") }}
     - {{ compiler("rust") }}
-    - cargo-bundle-licenses
+    - cargo-bundle-licenses 0.5.0
   host:
     - pip
     - python
     - maturin >=1,<2
     - typing-extensions  >=4.6.0,!=4.7.0
+    - wheel
```

Comment:
Does it fail without wheel? I assumed maturin built the wheel so it wasn't necessary to explicitly add it.

Reply:
wheel is not necessary here, the linter would need an update.

Reply:
Oh okay! I'll remove it! I was attempting to soothe the Linter God 🤣